{"code":"28P01","file":"auth.c","length":104,"level":"error","line":"326","message":"Error during Data Source initialization password authentication failed for user \"postgres\"","name":"error","routine":"auth_failed","severity":"FATAL","stack":"error: password authentication failed for user \"postgres\"\n    at Parser.parseErrorMessage (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:368:69)\n    at Parser.handlePacket (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:187:21)\n    at Parser.parse (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:102:30)\n    at Socket.<anonymous> (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/index.ts:7:48)\n    at Socket.emit (node:events:520:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"}
{"code":"3D000","file":"postinit.c","length":92,"level":"error","line":"1033","message":"Error during Data Source initialization database \"triplo\" does not exist","name":"error","routine":"InitPostgres","severity":"FATAL","stack":"error: database \"triplo\" does not exist\n    at Parser.parseErrorMessage (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:368:69)\n    at Parser.handlePacket (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:187:21)\n    at Parser.parse (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/parser.ts:102:30)\n    at Socket.<anonymous> (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/pg-protocol/src/index.ts:7:48)\n    at Socket.emit (node:events:520:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:191:23)"}
{"code":"42P07","driverError":{"code":"42P07","file":"index.c","length":90,"line":"900","name":"error","routine":"index_create","severity":"ERROR"},"file":"index.c","length":90,"level":"error","line":"900","message":"Error during Data Source initialization relation \"UQ_PHONE\" already exists","query":"CREATE TABLE \"user\" (\"id\" SERIAL NOT NULL, \"name\" character varying NOT NULL, \"email\" character varying NOT NULL, \"password\" character varying NOT NULL, \"phone\" character varying NOT NULL, \"updatedDate\" TIMESTAMP NOT NULL DEFAULT now(), \"createdDate\" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT \"UQ_PHONE\" UNIQUE (\"email\"), CONSTRAINT \"UQ_PHONE\" UNIQUE (\"phone\"), CONSTRAINT \"PK_cace4a159ff9f2512dd42373760\" PRIMARY KEY (\"id\"))","routine":"index_create","severity":"ERROR","stack":"QueryFailedError: relation \"UQ_PHONE\" already exists\n    at PostgresQueryRunner.query (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/driver/src/driver/postgres/PostgresQueryRunner.ts:331:19)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async PostgresQueryRunner.executeQueries (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/query-runner/src/query-runner/BaseQueryRunner.ts:660:13)\n    at async PostgresQueryRunner.createTable (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/driver/src/driver/postgres/PostgresQueryRunner.ts:610:9)\n    at async RdbmsSchemaBuilder.createNewTables (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/schema-builder/src/schema-builder/RdbmsSchemaBuilder.ts:635:13)\n    at async RdbmsSchemaBuilder.executeSchemaSyncOperationsInProperOrder (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/schema-builder/src/schema-builder/RdbmsSchemaBuilder.ts:225:9)\n    at async RdbmsSchemaBuilder.build (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/schema-builder/src/schema-builder/RdbmsSchemaBuilder.ts:95:13)\n    at async DataSource.synchronize (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/data-source/src/data-source/DataSource.ts:339:9)\n    at async DataSource.initialize (/Users/lukagurdulic/Desktop/lg_pp/triplo/node_modules/typeorm/data-source/src/data-source/DataSource.ts:277:43)\n    at async initializeDatabase (/Users/lukagurdulic/Desktop/lg_pp/triplo/packages/server/src/config/database.ts:19:3)"}
